frontend/components/MainPage/sections/AddFundsSection.tsx

```diff
@@ -101,92 +98,43 @@ export const OpenAddFundsSection = forwardRef<HTMLDivElement>((_, ref) => {
     [masterSafeAddress],
   );
 
-  useInterval(async () => {
```

Comment:
unsure this is needed here, this section should be static and not based on balances, assuming it was added for testing

Reply:
yeah, was a hack job for optimus POC, fine to remove